src/openvic-simulation/pop/Pop.cpp

```diff
@@ -157,26 +161,29 @@ bool PopManager::load_pop_type_file(
 	return ret;
 }
 
-bool PopManager::load_pop_into_province(Province& province, std::string_view pop_type_identifier, ast::NodeCPtr pop_node)
-	const {
-	PopType const* type = get_pop_type_by_identifier(pop_type_identifier);
+bool PopManager::load_pop_into_vector(
+	RebelManager const& rebel_manager, std::vector<Pop>& vec, PopType const& type, ast::NodeCPtr pop_node
+) const {
 	Culture const* culture = nullptr;
 	Religion const* religion = nullptr;
-	Pop::pop_size_t size = 0;
+	fixed_point_t size = 0; /* Some genius filled later start dates with non-integer sized pops */
+	fixed_point_t militancy = 0, consciousness = 0;
+	RebelType const* rebel_type = nullptr;
+
 	bool ret = expect_dictionary_keys(
 		"culture", ONE_EXACTLY, culture_manager.expect_culture_identifier(assign_variable_callback_pointer(culture)),
 		"religion", ONE_EXACTLY, religion_manager.expect_religion_identifier(assign_variable_callback_pointer(religion)),
-		"size", ONE_EXACTLY, expect_uint(assign_variable_callback(size)),
-		"militancy", ZERO_OR_ONE, success_callback,
-		"rebel_type", ZERO_OR_ONE, success_callback
+		"size", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(size)),
```

Comment:
Decimal sizes are accepted when loading data but are truncated in game.
Should it log a warning when size % 1 > 0 ?

Reply:
Doing this for each pop individually would cause too much warning spam, since hundreds if not thousands of pops have non-integer sizes in 1861. We could easily reduce it to one warning per file, but ideally it'd be a single warning if any pops in the start date have non-integer sizes. I'll add this later today.